platformio.ini

```diff
@@ -96,6 +96,7 @@ lib_deps =
   https://github.com/meshtastic/ArduinoThread.git#1ae8778c85d0a2a729f989e0b1e7d7c4dc84eef0
   nanopb/Nanopb@^0.4.9
   erriez/ErriezCRC32@^1.0.1
+  fmtlib/fmt@^8.1.1
```

Comment:
I know nothing about these things, but I noticed on the release page for the library its zip was >1MB.

Reply:
95% of that is documentation. the lib itself is about 20 small sourcecode files. And the dynamic linker will throw out most of that.

This lib is included in C++20, since we use C++11 the dependency is neccessary.